### DISCUSSION TITLE
Daily crashes with AIRDCPP

### DISCUSSION THREAD

**Discussion by mandrix1** [airdcpp/airdcpp-windows#134] [Q&A]
Could someone look at this error log and tell me what the heck is going on?
[exceptioninfo.txt](https://github.com/user-attachments/files/16763380/exceptioninfo.txt)

**Comment by maksis** [comment]
If your system isn't running out of memory, please post an issue with the required information: https://github.com/airdcpp/airdcpp-windows/issues (discussions aren't meant for crashes)
